packages/core/src/lib/template/template.interface.ts

```diff
@@ -25,6 +26,7 @@ export enum ChannelTypeEnum {
 }
 
 export interface ITriggerPayload {
+  $providerId?: string[];
```

Comment:
Not sure I understand the necessity of this field here. Why is this the responsibility of the trigger payload to specify the providerId? Shouldn't the trigger only specify the communication credentials? For example the slack token or onesignal token